### ISSUE TITLE
best practices and seo errors in AnchorElements on some pages

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#10907]
![image](https://user-images.githubusercontent.com/4071474/83722649-3d757d00-a5f2-11ea-877c-2a5af2601750.png)

https://www.coursehero.com/sg/introduction-to-psychology/historical-and-modern-psychological-perspectives/

bisected to https://github.com/GoogleChrome/lighthouse/pull/10662

```
Required AnchorElements gatherer encountered an error: Protocol error (DOM.pushNodeByPathToFrontend): No node with given path found
```

**Comment by connorjclark** [comment]
maybe we should run LH on a corpus of a few hundred URLs before every release (or in CI?) so we can catch more of these types of issues on new gatherers / updating axe / etc.

**Comment by patrickhulce** [comment]
FYI dupe of https://github.com/GoogleChrome/lighthouse/issues/10873#issuecomment-636026607 fixed by #10877 :)

**Comment by patrickhulce** [comment]
this was fixed in 6.1 :) #10877
